README.md

```diff
@@ -25,6 +25,26 @@ For the Alpha Release, the **API** documented here are supporting both the MyRa
 
 ## Usage
 
+### Environment variables
+
+Before being able to run the API and/or tests, you need to setup some environment variables. Copy and paste this into a `.env` file:
+```
+POSTGRESQL_DATABASE=myra
+POSTGRESQL_DATABASE_TEST=myra_test
+POSTGRESQL_HOST=db
+POSTGRESQL_PORT=5432
+
+PROJECT=myra_range
+ENVIRONMENT=development
+API_PORT=8080
+BUILD_TARGET=base
+
+POSTGRESQL_PASSWORD= banana
```

Comment:
This causes tests to fail

Reply:
@micheal-w-wells Do you mean the PR test run fails? That would be because of https://github.com/bcgov/range-api/issues/175. 

Or do you mean these environment variables don't work for you?

Reply:
i cloned the repo again followed the steps with exception of copying this file as it wasnt in dev, and it fails when it runs tests on password auth

Reply:
Can you elaborate on what you mean by "exception of copying this file"? Did you copy these environment variables into a `.env` file?

Reply:
What i mean is i was working off dev, not this branch.  I went to this PR and copied this .env down.

Reply:
Oh gotcha, that shouldn't be an issue. Are all of your tests failing with the same error? Maybe paste some logs.

 It's also possible that even though the project is in a different directory, docker-compose is still using the existing containers. Maybe try a `docker-compose down -v` (be aware this deletes your volumes, so you may want to back up your local database if you don't want to lose it).

Reply:
k when i get time i'll do a full run through and make sure to close running stuff.

Reply:
I've added a note that environment variables have to be sourced in your shell. When you get the chance, let me know if that fixes this for you.